src/header.js

```diff
@@ -212,51 +218,37 @@ this.Header = function Header({
   });
 
   let loadChangedHandlerId = null;
-  let resourceLoadStartedHandlerId = null;
+  let backForwardListChangedHandlerId = null;
   state.notify("webview", (webview, previous) => {
+    //on tab change
+
+    log("hi");
+
+    const backForwardList = webview.get_back_forward_list();
+
+    //disconnect handlers of other tab
     if (previous) {
+      const lastBackForwardList = previous.get_back_forward_list();
```

Comment:
can you move this line into the `if (backForwardListChangedHandlerId) {` since it's not used somehwere else